src/db/graph/queries/get.rs

```diff
@@ -236,6 +236,56 @@ pub fn user_tags(user_id: &str) -> neo4rs::Query {
     .param("user_id", user_id)
 }
 
+pub fn get_viewer_trusted_network_tags(user_id: &str, viewer_id: &str, depth: u8) -> neo4rs::Query {
+    // Initialize the cypher query
+    let mut cypher = String::new();
+
+    // Create the viewer trusted network
+    cypher.push_str("MATCH (viewer:User {id: $viewer_id})\n");
+
+    let mut call_statements = Vec::new();
+
+    // Dynamically generate Cypher for each depth level
+    for distance in 1..=depth {
+        let depth_match = if distance == 1 {
+            "(viewer)-[:FOLLOWS]->(tagger:User)".to_string()
+        } else {
+            format!(
+                "(viewer)-[:FOLLOWS]->{}(tagger:User)",
+                "(:User)-[:FOLLOWS]->".repeat(distance as usize - 1)
+            )
+        };
+
+        call_statements.push(format!(
+            "WITH viewer MATCH {} RETURN tagger AS potentialTagger",
+            depth_match
+        ));
+    }
+
+    // Combine the CALL block with UNION
+    let call_clause = format!("CALL {{\n{}\n}}\n", call_statements.join("\nUNION\n"));
+
+    cypher.push_str(&call_clause);
```

Comment:
Looks overly complex. Cypher is always the same regardless of depth level. We can use the `[:FOLLOWS*1..depth]` to match taggers at any depth with simple cypher syntax, right? Could we maybe just replace these 20 lines for this for something like this? Or is the behaviour different?
```rust 
query = format!("(viewer)-[:FOLLOWS*1..{}]->(tagger:User)", depth);
```